apk/org.wso2.apk.apimgt.impl/src/main/java/org/wso2/apk/apimgt/impl/ZIPUtils.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.apk.apimgt.impl;
+
+import org.wso2.carbon.apimgt.api.APIManagementException;
```

Comment:
Should be org.wso2.apk

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/11571/commits/d257b9f10d19f43b8e40d3e5ae67a217a6b50a20